recipes/psychopy/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set name = "psychopy" %}
+{% set pypi_name = "PsychoPy" %}
```

No need for the duplication, the name is explicitly converted to lowercase with the jinja2 filter (`{{ name|lower }}`):
```suggestion
{% set name = "PsychoPy" %}
```

---

I did it this way because I would like the `conda-forge` package to be called `psychopy` (all-lowercase) and not `PsychoPy`. Or do you think this is a bad idea?

---

Here `name` is just a variable name, the actual package name comes from:
```yaml
package:
  name: {{ name|lower }}
  version: {{ version }}
```
and it's explicitly converted to lowercase. It could just as easily be `name: {{ pypi_name|lower }}` but using `name` for the variable is more consistent with the rest of conda-forge.

---

Got you. Thanks!